src/transport/smtp/client/async_net.rs

```diff
@@ -44,28 +44,42 @@ use crate::transport::smtp::client::net::resolved_address_filter;
 use crate::transport::smtp::{error, Error};
 
 /// A network stream
+#[derive(Debug)]
 pub struct AsyncNetworkStream {
     inner: InnerAsyncNetworkStream,
 }
 
+#[cfg(feature = "tokio1")]
+pub trait AsyncStream: AsyncRead + AsyncWrite + Send + Sync + Unpin + fmt::Debug {
```

Comment:
I'm not sure about the name, as it would conflict with a potential future async-std version of this trait

Reply:
Changed to `AsyncTokioStream`. But open to better suggestions. :)